catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -492,8 +492,9 @@ private void cloneSpriteVariables(Scene currentScene, Sprite cloneSprite) {
 	}
 
 	private void cloneLook(Sprite cloneSprite) {
-		if (cloneSprite.getLookDataList().size() > 0) {
-			cloneSprite.look.setLookData(cloneSprite.getLookDataList().get(0));
+		int currentLookDataIndex = this.lookList.indexOf(this.look.getLookData());
+		if (currentLookDataIndex != -1) {
+			cloneSprite.look.setLookData(cloneSprite.lookList.get(currentLookDataIndex));
```

Comment:
Both, the sprite and cloneSprite have an identical lookList. Same length, same order, same (cloned!) elements.